doc/source/Tutorials/GettingStarted/index.md

```diff
@@ -1,407 +1,499 @@
+
+Next, pin the same node to the _Scene Hierarchy_. This time, use the middle-click shortcut:
+
+1. Select the InteractiveAppleseedRender node.
+
+2. Middle-click and drag the InteractiveAppleseedRender node from the _Graph Editor_ into the
+  _Scene Hierarchy_.
+
+As with the _Viewer_, the _Scene Hierarchy_ will now remain locked to the output of InteractiveAppleseedRender, regardless of your selection. Now you are free to select any node for editing in the _Node Editor_, but you will always be viewing the final results of the last node in the graph.
+
+For the final adjustment to the UI, create another _Viewer_ in the top-left panel, and pin the Catalogue node to it:
+
+1. At the top-right of the top panel, click ![the layout menu button](images/layoutButton.png "Layout menu button") to open the layout menu.
+
+2. Select _Viewer_. A new _Viewer_ will appear on the panel next to the first one.
+
+3. Middle-click and drag the Catalogue node onto the new _Viewer_. That _Viewer_ is now pinned to that node.
+
+Now you can switch between the scene's geometry (first _Viewer_) and the rendered image (second _Viewer_). Now it is time to shade Gaffy.
+
+
+## Adding Shaders and Lighting ##
+
+It's time to add shaders and lighting. It will help to think of a graph as being composed of three distinct phases: generating the geometry, applying the lighting and shading, and rendering. Lights act like another scene object, so you can group one into the scene like the other nodes. The shaders, however, are different: in order for the render option nodes to apply to the geometry and to inherit the shaders, you will need to add them somewhere between the scene-creating nodes and render options nodes.
+
+
+### Making Some Space ###
+
+Since you will be adding shaders nodes between the scene nodes and the render options nodes, you will first need to add some space in the graph.
+
+To create some empty space in the centre of the graph:
+
+1. Select the lower five nodes by clicking and dragging a marquee over them.
+
+2. Click and drag the nodes to a lower position in the graph.
+
+    ![The graph with some added space](images/mainRenderSettingsWithGap.png "The graph with some added space")
+
+
+### Adding a Shader ###
+
+Now that you have more space, it's time to add some shading nodes:
+
+1. Below and to the left of the Group node, create a Disney Material node (_Appleseed_ > _Shader_ > _Material_ > *As_Disney_Material*).
+
+2. In the _Node Editor_, give the shader some reflective surface properties:
+	- Set the Specular plug to `0.6`.
+	- Set the Roughness plug to `0.35`.
+
+3. With the Disney Material node still selected, create a ShaderAssignment node (_Scene_ > _Attributes_ > _ShaderAssignment_).
+
+4. Click and drag the ShaderAssignment node onto the connector connecting the Group and StandardOptions nodes. The ShaderAssignment_node will be interjected between them.
```

Comment:
"onto the connector connecting the Group and StandardOptions" is quite a brain twister... I stumbled over it 3 times in a row. How about "onto the connector _**between**_ the Group and StandardOptions" ?